src/guiguts/application.py

```diff
@@ -332,6 +332,16 @@ def initialize_preferences(self) -> None:
             ),
         )
         preferences.set_default(PrefKey.TEAROFF_MENUS, False)
+        preferences.set_default(PrefKey.TEXT_FONT_FAMILY, "Courier")
+        preferences.set_callback(
+            PrefKey.TEXT_FONT_FAMILY,
+            lambda *value: maintext().set_font(),
+        )
+        preferences.set_default(PrefKey.TEXT_FONT_SIZE, 10)
```

Comment:
Just curious -- why set the default to font size to 10?

Reply:
Originally because it's Courier 10 in GG1. 
Now I've changed the default to be DPSM if you have it installed, then 10 is actually quite reasonable for that font, whereas it's a little small for Courier. The user only has to change it once if they don't like it. If you make it 12, for example, then I for one, would not want that with DPSM. 🤷